manifests/filebeat.pp

```diff
@@ -32,10 +35,31 @@
     notify  => Service['filebeat']
   }
 
+  exec { 'Installing filebeat module ... Downloading package':
+   path    => '/usr/bin',
+   command => "curl -o /root/${$wazuh_module_filebeat_module} https://packages-dev.wazuh.com/3.x/filebeat/${$wazuh_module_filebeat_module}",
+  }
+
+  exec { 'Unpackaging ...':
+    command => "/bin/tar -xzvf /root/wazuh-filebeat-0.1.tar.gz -C /usr/share/filebeat/module",
+    notify  => Service['filebeat']
+  }
+
+  class directory_tree {
```

Comment:
It's not required to create a class for this, simply:

```
    file { '/usr/share/filebeat/module/wazuh':
      ensure => 'directory',
      mode   => '0755',
    }
```

Should work

Reply:
Hi Jose!

Thanks for your attention. Check the changes [here](https://github.com/wazuh/wazuh-puppet/commit/1301d408b350d150f40ea1e58dbace59119cc99a).

Kind regards,

Rshad